docker-compose.yml

```diff
@@ -53,27 +50,6 @@ services:
       PGDATA: /var/lib/postgresql/data/pgdata/mydata
     ports:
       - 5432:5432
-  search:
```

Comment:
Remove pipeline-related services from docker-compose configuration because not used or needed.